frontend/src/store/modules/userList/actions.js

```diff
@@ -1,11 +1,14 @@
 import httpService from '../../../services/common/httpService';
 
 export default {
-    getListsByUser: (context, userId) => {
+    getListsByUser: (context, userId = 'self') => {
         return new Promise((resolve, reject) => {
-            httpService.get('/user-list')
-                .then(function (result) {
-                    resolve(result.data.data.filter(list => list.user_id === userId));
+            let route = '/users/'+userId+'/lists';
+            httpService.get(route)
+                .then(function (response) {
+                    const userLists = response.data.data;
+                    context.commit('SET_USER_LISTS', userLists);
```

Comment:
userLists should have new state structure { byId: { "1": {}, "2" }, allIds: [] }. Use noramilzerService